Nest JS basic script error

basic npm install command is `npm install`
but now, setting is `npm run install`
so, make some error

```
04:31:34.718  Step 4/6 : RUN npm run install
04:31:34.737   ---> Running in c9f7fe9bfc34
04:31:35.592  npm`
04:31:35.592   ERR! Missing script: "install"npm ERR! 
npm ERR!
04:31:35.593   Did you mean this?
04:31:35.593  npm
04:31:35.593   ERR!     npm uninstall # Remove a packagenpm
04:31:35.593   ERR!
04:31:35.593   npm ERR! To see a list of scripts, run:
npm
04:31:35.593   ERR!   npm run
04:31:35.594  
04:31:35.594  npm ERR!`
```

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
 	"name": "coolify",
 	"description": "An open-source & self-hostable Heroku / Netlify alternative.",
-	"version": "2.0.22",
+	"version": "2.0.23",
 	"license": "AGPL-3.0",
 	"scripts": {
 		"dev": "docker-compose -f docker-compose-dev.yaml up -d && NODE_ENV=development svelte-kit dev --host 0.0.0.0",

--- a/src/lib/buildPacks/gatsby.ts
+++ b/src/lib/buildPacks/gatsby.ts
@@ -7,6 +7,7 @@ const createDockerfile = async (data, imageforBuild): Promise<void> => {
 
 	Dockerfile.push(`FROM ${imageforBuild}`);
 	Dockerfile.push('WORKDIR /usr/share/nginx/html');
+	Dockerfile.push(`LABEL coolify.image=true`);
 	Dockerfile.push(`COPY --from=${applicationId}:${tag}-cache /usr/src/app/${publishDirectory} ./`);
 	Dockerfile.push(`EXPOSE 80`);
 	Dockerfile.push('CMD ["nginx", "-g", "daemon off;"]');

--- a/src/lib/buildPacks/nestjs.ts
+++ b/src/lib/buildPacks/nestjs.ts
@@ -7,6 +7,7 @@ const createDockerfile = async (data, image): Promise<void> => {
 
 	Dockerfile.push(`FROM ${image}`);
 	Dockerfile.push('WORKDIR /usr/src/app');
+	Dockerfile.push(`LABEL coolify.image=true`);
 	Dockerfile.push(
 		`COPY --from=${applicationId}:${tag}-cache /usr/src/app/${baseDirectory || ''} ./`
 	);

--- a/src/lib/buildPacks/nextjs.ts
+++ b/src/lib/buildPacks/nextjs.ts
@@ -16,6 +16,7 @@ const createDockerfile = async (data, image): Promise<void> => {
 
 	Dockerfile.push(`FROM ${image}`);
 	Dockerfile.push('WORKDIR /usr/src/app');
+	Dockerfile.push(`LABEL coolify.image=true`);
 	if (secrets.length > 0) {
 		secrets.forEach((secret) => {
 			if (secret.isBuildSecret) {

--- a/src/lib/buildPacks/node.ts
+++ b/src/lib/buildPacks/node.ts
@@ -16,6 +16,7 @@ const createDockerfile = async (data, image): Promise<void> => {
 
 	Dockerfile.push(`FROM ${image}`);
 	Dockerfile.push('WORKDIR /usr/src/app');
+	Dockerfile.push(`LABEL coolify.image=true`);
 	if (secrets.length > 0) {
 		secrets.forEach((secret) => {
 			if (secret.isBuildSecret) {

--- a/src/lib/buildPacks/nuxtjs.ts
+++ b/src/lib/buildPacks/nuxtjs.ts
@@ -16,6 +16,7 @@ const createDockerfile = async (data, image): Promise<void> => {
 
 	Dockerfile.push(`FROM ${image}`);
 	Dockerfile.push('WORKDIR /usr/src/app');
+	Dockerfile.push(`LABEL coolify.image=true`);
 	if (secrets.length > 0) {
 		secrets.forEach((secret) => {
 			if (secret.isBuildSecret) {

--- a/src/lib/buildPacks/php.ts
+++ b/src/lib/buildPacks/php.ts
@@ -6,6 +6,7 @@ const createDockerfile = async (data, image): Promise<void> => {
 	const Dockerfile: Array<string> = [];
 
 	Dockerfile.push(`FROM ${image}`);
+	Dockerfile.push(`LABEL coolify.image=true`);
 	Dockerfile.push('RUN a2enmod rewrite');
 	Dockerfile.push('WORKDIR /var/www/html');
 	Dockerfile.push(`COPY ./${baseDirectory || ''} /var/www/html`);

--- a/src/lib/buildPacks/react.ts
+++ b/src/lib/buildPacks/react.ts
@@ -6,6 +6,7 @@ const createDockerfile = async (data, image): Promise<void> => {
 	const Dockerfile: Array<string> = [];
 
 	Dockerfile.push(`FROM ${image}`);
+	Dockerfile.push(`LABEL coolify.image=true`);
 	Dockerfile.push('WORKDIR /usr/share/nginx/html');
 	Dockerfile.push(`COPY --from=${applicationId}:${tag}-cache /usr/src/app/${publishDirectory} ./`);
 	Dockerfile.push(`EXPOSE 80`);

--- a/src/lib/buildPacks/rust.ts
+++ b/src/lib/buildPacks/rust.ts
@@ -8,6 +8,7 @@ const createDockerfile = async (data, image, name): Promise<void> => {
 	const Dockerfile: Array<string> = [];
 	Dockerfile.push(`FROM ${image}`);
 	Dockerfile.push('WORKDIR /usr/src/app');
+	Dockerfile.push(`LABEL coolify.image=true`);
 	Dockerfile.push(`COPY --from=${applicationId}:${tag}-cache /usr/src/app/target target`);
 	Dockerfile.push(`COPY --from=${applicationId}:${tag}-cache /usr/local/cargo /usr/local/cargo`);
 	Dockerfile.push(`COPY . .`);

--- a/src/lib/buildPacks/static.ts
+++ b/src/lib/buildPacks/static.ts
@@ -16,6 +16,7 @@ const createDockerfile = async (data, image): Promise<void> => {
 
 	Dockerfile.push(`FROM ${image}`);
 	Dockerfile.push('WORKDIR /usr/share/nginx/html');
+	Dockerfile.push(`LABEL coolify.image=true`);
 	if (secrets.length > 0) {
 		secrets.forEach((secret) => {
 			if (secret.isBuildSecret) {

--- a/src/lib/buildPacks/svelte.ts
+++ b/src/lib/buildPacks/svelte.ts
@@ -7,6 +7,7 @@ const createDockerfile = async (data, image): Promise<void> => {
 
 	Dockerfile.push(`FROM ${image}`);
 	Dockerfile.push('WORKDIR /usr/share/nginx/html');
+	Dockerfile.push(`LABEL coolify.image=true`);
 	Dockerfile.push(`COPY --from=${applicationId}:${tag}-cache /usr/src/app/${publishDirectory} ./`);
 	Dockerfile.push(`EXPOSE 80`);
 	Dockerfile.push('CMD ["nginx", "-g", "daemon off;"]');

--- a/src/lib/buildPacks/vuejs.ts
+++ b/src/lib/buildPacks/vuejs.ts
@@ -7,6 +7,7 @@ const createDockerfile = async (data, image): Promise<void> => {
 
 	Dockerfile.push(`FROM ${image}`);
 	Dockerfile.push('WORKDIR /usr/share/nginx/html');
+	Dockerfile.push(`LABEL coolify.image=true`);
 	Dockerfile.push(`COPY --from=${applicationId}:${tag}-cache /usr/src/app/${publishDirectory} ./`);
 	Dockerfile.push(`EXPOSE 80`);
 	Dockerfile.push('CMD ["nginx", "-g", "daemon off;"]');

--- a/src/lib/components/templates.ts
+++ b/src/lib/components/templates.ts
@@ -1,7 +1,7 @@
 function defaultBuildAndDeploy(packageManager) {
 	return {
 		installCommand:
-			packageManager === 'npm' ? `${packageManager} run install` : `${packageManager} install`,
+			packageManager === 'npm' ? `${packageManager} install` : `${packageManager} install`,
 		buildCommand:
 			packageManager === 'npm' ? `${packageManager} run build` : `${packageManager} build`,
 		startCommand:

--- a/src/lib/docker.ts
+++ b/src/lib/docker.ts
@@ -19,6 +19,7 @@ export async function buildCacheImageWithNode(data, imageForBuild) {
 	const Dockerfile: Array<string> = [];
 	Dockerfile.push(`FROM ${imageForBuild}`);
 	Dockerfile.push('WORKDIR /usr/src/app');
+	Dockerfile.push(`LABEL coolify.image=true`);
 	if (secrets.length > 0) {
 		secrets.forEach((secret) => {
 			if (secret.isBuildSecret) {

--- a/src/lib/haproxy/index.ts
+++ b/src/lib/haproxy/index.ts
@@ -620,6 +620,9 @@ export async function configureSimpleServiceProxyOn({ id, domain, port }) {
 }
 
 export async function configureSimpleServiceProxyOff(fqdn) {
+	if (!fqdn) {
+		return;
+	}
 	const domain = getDomain(fqdn);
 	const haproxy = await haproxyInstance();
 	await checkHAProxy(haproxy);

--- a/src/lib/queues/builder.ts
+++ b/src/lib/queues/builder.ts
@@ -239,10 +239,16 @@ export default async function (job) {
 			baseDirectory,
 			publishDirectory
 		});
+		let envFound = false;
+		try {
+			envFound = !!(await fs.stat(`${workdir}/.env`));
+		} catch (error) {
+			//
+		}
 		try {
 			saveBuildLog({ line: 'Deployment started.', buildId, applicationId });
 			const { stderr } = await asyncExecShell(
-				`DOCKER_HOST=${host} docker run --env-file=${workdir}/.env ${labels.join(
+				`DOCKER_HOST=${host} docker run ${envFound && `--env-file=${workdir}/.env`} ${labels.join(
 					' '
 				)} --name ${imageId} --network ${
 					docker.network

--- a/src/lib/queues/cleanup.ts
+++ b/src/lib/queues/cleanup.ts
@@ -36,7 +36,7 @@ export default async function () {
 			console.log(error);
 		}
 		if (!dev) {
-			//Cleanup images that are not managed by coolify
+			// Cleanup images that are not managed by coolify
 			try {
 				await asyncExecShell(
 					`DOCKER_HOST=${host} docker image prune --filter 'label!=coolify.image=true' -a -f`
@@ -44,12 +44,12 @@ export default async function () {
 			} catch (error) {
 				console.log(error);
 			}
-		}
-		// Cleanup dangling images
-		try {
-			await asyncExecShell(`DOCKER_HOST=${host} docker image prune -f`);
-		} catch (error) {
-			console.log(error);
+			// Cleanup old images >3 days
+			try {
+				await asyncExecShell(`DOCKER_HOST=${host} docker image prune --filter "until=72h" -a -f`);
+			} catch (error) {
+				console.log(error);
+			}
 		}
 	}
 }

--- a/src/routes/applications/[id]/__layout.svelte
+++ b/src/routes/applications/[id]/__layout.svelte
@@ -128,7 +128,7 @@
 					title="Stop application"
 					type="submit"
 					disabled={!$session.isAdmin}
-					class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 hover:bg-green-600 hover:text-white"
+					class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 text-red-500"
 					data-tooltip={$session.isAdmin
 						? 'Stop application'
 						: 'You do not have permission to stop the application.'}
@@ -153,7 +153,7 @@
 						title="Rebuild application"
 						type="submit"
 						disabled={!$session.isAdmin}
-						class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 hover:bg-green-600 hover:text-white"
+						class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 hover:text-green-500"
 						data-tooltip={$session.isAdmin
 							? 'Rebuild application'
 							: 'You do not have permission to rebuild application.'}

--- a/src/routes/applications/[id]/__layout.svelte
+++ b/src/routes/applications/[id]/__layout.svelte
@@ -182,7 +182,7 @@
 						title="Build and start application"
 						type="submit"
 						disabled={!$session.isAdmin}
-						class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 hover:bg-green-600 hover:text-white"
+						class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 text-green-500"
 						data-tooltip={$session.isAdmin
 							? 'Build and start application'
 							: 'You do not have permission to Build and start application.'}

--- a/src/routes/databases/[id]/__layout.svelte
+++ b/src/routes/databases/[id]/__layout.svelte
@@ -120,7 +120,7 @@
 					title="Stop database"
 					type="submit"
 					disabled={!$session.isAdmin}
-					class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 hover:bg-purple-600 hover:text-white"
+					class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 text-red-500"
 					data-tooltip={$session.isAdmin
 						? 'Stop database'
 						: 'You do not have permission to stop the database.'}
@@ -146,7 +146,7 @@
 					title="Start database"
 					type="submit"
 					disabled={!$session.isAdmin}
-					class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 hover:bg-purple-600 hover:text-white"
+					class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 text-green-500"
 					data-tooltip={$session.isAdmin
 						? 'Start database'
 						: 'You do not have permission to start the database.'}

--- a/src/routes/login/index.svelte
+++ b/src/routes/login/index.svelte
@@ -71,12 +71,14 @@
 						class:text-stone-600={loading}
 						class:bg-coollabs={!loading}>{loading ? 'Authenticating...' : 'Login'}</button
 					>
+
 					<button
 						on:click|preventDefault={() => goto('/register')}
-						class="hover:opacity-90 text-white">Register</button
+						class="bg-transparent hover:bg-coolgray-300	text-white ">Register</button
 					>
-					<button class="bg-transparent" on:click|preventDefault={() => goto('/reset')}
-						>Reset password</button
+					<button
+						class="bg-transparent hover:bg-coolgray-300"
+						on:click|preventDefault={() => goto('/reset')}>Reset password</button
 					>
 				</div>
 			</form>

--- a/src/routes/services/[id]/__layout.svelte
+++ b/src/routes/services/[id]/__layout.svelte
@@ -140,7 +140,7 @@
 					title="Stop Service"
 					type="submit"
 					disabled={!$session.isAdmin}
-					class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 hover:bg-pink-600 hover:text-white"
+					class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 text-red-500"
 					data-tooltip={$session.isAdmin
 						? 'Stop Service'
 						: 'You do not have permission to stop the service.'}
@@ -166,7 +166,7 @@
 					title="Start Service"
 					type="submit"
 					disabled={!$session.isAdmin}
-					class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 hover:bg-pink-600 hover:text-white"
+					class="icons bg-transparent tooltip-bottom text-sm flex items-center space-x-2 text-green-500"
 					data-tooltip={$session.isAdmin
 						? 'Start Service'
 						: 'You do not have permission to start the service.'}

